ArduPlane/mode_takeoff.cpp

```diff
@@ -164,4 +160,9 @@ void ModeTakeoff::navigate()
     // Zero indicates to use WP_LOITER_RAD
     plane.update_loiter(0);
 }
-
+#if AP_FENCE_ENABLED
+void ModeTakeoff::_exit()
+{
+    plane.fence.auto_enable_fence_after_takeoff();
```

Comment:
i think this should only be if armed and flying

Reply:
donce